tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -3734,6 +3735,21 @@ public Print print(Operand<TString> input, Print.Options... options) {
     return Prod.create(scope, input, axis, options);
   }
 
+  /**
+   *
+   * @param <T> data type for {@code output()} output
+   * @param values
+   * @param axis
+   * @param inputMins
+   * @param inputMaxes
+   * @return a new instance of QuantizedConcatV2
+   */
+  public <T extends TType, U extends TNumber> QuantizedConcatV2<T> quantizedConcatV2(
+      Iterable<Operand<T>> values, Operand<U> axis, Iterable<Operand<TFloat32>> inputMins,
+      Iterable<Operand<TFloat32>> inputMaxes) {
+    return QuantizedConcatV2.create(scope, values, axis, inputMins, inputMaxes);
+  }
```

Comment:
That's a new op that needs to be reclassified to the right package. In fact, it should replace the actual `QuantizedConcat` op that is found under `org.tensorflow.op.quantization`. 

Here's how to do it:
* Duplicate the file [`api_def_QuantizedConcat.pbtxt`](https://github.com/tensorflow/java/blob/master/tensorflow-core/tensorflow-core-api/src/bazel/api_def/api_def_QuantizedConcat.pbtxt) to `api_def_QuantizedConcatV2.pbtxt` in the same folder.
* Replace the `endpoint` in the old file by `visibility: SKIP`, like in [this one](https://github.com/tensorflow/java/blob/master/tensorflow-core/tensorflow-core-api/src/bazel/api_def/api_def_PriorityQueue.pbtxt)
* Replace the `graph_op_name` in the new file by `QuantizedConcatV2`

Reply:
Could you push that to my branch? Or I'll just leave those changes out and you can do it before or after we merge this PR?

Reply:
Yeah I can do that